From 248c95b6dd2fe65a0e359e98a9a2d3d7da1bbb2e Mon Sep 17 00:00:00 2001
From: Niall Pemberton <niallp@apache.org>
Date: Sun, 9 Jan 2011 00:29:01 +0000
Subject: [PATCH] Port LANG-635 to LANG 2.x Branch - fixing the pass along of
 compareTransients to the reflectionCompare method as identified by Robert
 Radkiewicz

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/lang/branches/LANG_2_X@1056843 13f79535-47bb-0310-9956-ffa450edef68
---
 .../java/org/apache/commons/lang/builder/CompareToBuilder.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/java/org/apache/commons/lang/builder/CompareToBuilder.java b/src/main/java/org/apache/commons/lang/builder/CompareToBuilder.java
index 2352969a0e2..28411d3e75d 100644
--- a/src/main/java/org/apache/commons/lang/builder/CompareToBuilder.java
+++ b/src/main/java/org/apache/commons/lang/builder/CompareToBuilder.java
@@ -272,7 +272,7 @@ public static int reflectionCompare(Object lhs, Object rhs, String[] excludeFiel
     public static int reflectionCompare(Object lhs, Object rhs, boolean compareTransients, 
                                         Class reflectUpToClass) 
     {
-        return reflectionCompare(lhs, rhs, false, reflectUpToClass, null);
+        return reflectionCompare(lhs, rhs, compareTransients, reflectUpToClass, null);
     }
 
     /**
